### DISCUSSION TITLE
[Wallet] Sovereign

### DISCUSSION THREAD

**Discussion by jennijuju** [filecoin-project/community#62] [Project Showcase]
Learn about Sovereign, built by Applications Research Group (ARG), [here](https://github.com/filecoin-project/community/blob/master/projects/wallet/sovereign.md).

Team:
@jimmylee @whyrusleeping

**Comment by jimmylee** [comment]
## Previews

![ecosystem-1](https://user-images.githubusercontent.com/310223/108011753-a18f7400-6fbc-11eb-926c-c7104e0fd36c.png)

## Updates
For the week ending on Friday, Feb 12th

* We gave the Filecoin wallet the name Sovereign.
* Because of demand, I open sourced the wallet. 
  * **I don't recommend anyone use this wallet unless they are 100% sure they know what they are doing**. I am hesitant to share because @whyrusleeping (Protocol Labs) and I are still experimenting. Regardless here it is: https://github.com/slate-engineering/wallet
* Both @raulk and @dignifiedquire have provided feedback! Thank you, we will add currency + a better design around fees and the Filecoin you are sending ASAP.
* We're working through the remaining issues here before packaging a DMG: https://github.com/slate-engineering/wallet/issues

## Social
Notable social posts for the week.

* https://twitter.com/wwwjim/status/1358859787643932674
* https://twitter.com/wwwjim/status/1358627448124887043
* https://twitter.com/wwwjim/status/1357260210729984000
* https://twitter.com/wwwjim/status/1358352913647558661

**Comment by jimmylee** [comment]
## Previews

<img width="1455" alt="Screen Shot 2021-02-19 at 7 29 54 PM" src="https://user-images.githubusercontent.com/310223/108582543-e4b55400-72e8-11eb-97b9-a663e314920f.png">

<img width="803" alt="Screen Shot 2021-02-19 at 7 31 06 PM" src="https://user-images.githubusercontent.com/310223/108582567-08789a00-72e9-11eb-925f-95d1c0a547ba.png">

## Updates
For the week ending on Friday, Feb 19th

* We shipped version 0.0.3, this version is stable!
* You can download it here: https://slate.host/slate/filecoin
* Supports real USD values now using an IEXCloud endpoint. Thank you @johannes-jp for the recommendation.
